icu4c/source/common/loclikely.cpp

```diff
@@ -1111,22 +1145,31 @@ _uloc_minimizeSubtags(const char* localeID,
                         langLength,
                         script,
                         scriptLength,
-                        NULL,
+                        nullptr,
                         0,
                         trailing,
                         trailingLength,
-                        sink,
```

Comment:
for example, how easy for you to figure out which "sink " this sink is referring to? How easy for you to spot this "sink" is not the "sink" in line 1119, not the "sink" in line 1073, not the sink in line 993 but the "sink" passed in as input parameter.